src/containers/chat/chat-header/ChatHeader.tsx

```diff
@@ -76,6 +96,35 @@ const ChatHeader: FC<ChatHeaderProps> = ({
     setIsSearchOpen(false)
   }
 
+  const getAllMessages = useCallback(
+    () =>
+      messageService.getMessages({
+        chatId: currentChat._id,
+        limit: 100,
```

Comment:
why do you fetch only 100 messages? `Number.MAX_SAFE_INTEGER` will be better I guess

Reply:
fixed